features/step_definitions/tag_set_steps.rb

```diff
@@ -140,4 +140,10 @@
 When /^I view the tag set "([^\"]*)"/ do |tagset|
   tagset = OwnedTagSet.find_by_title!(tagset)
   visit tag_set_path(tagset)
-end
\ No newline at end of file
+end
+
+When(/^I flush the wrangling sidebar caches$/) do
```

Comment:
This issue appears to be broken on test. When you submit a fix for it, can you please move this step definition into the proper file? Right now it's in the tag _set_ steps, but it's not related to tag sets -- it should be in https://github.com/otwcode/otwarchive/blob/master/features/step_definitions/tag_steps.rb, which is where wrangling stuff lives. Thanks!